doc/proposals/ansible-helm-addapi.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Ansible/Helm add api command  Proposal for Operator SDK
+authors:
+  - "@bharathi-tenneti"
+reviewers:
+  - "@fabianvf"
+  - "@cmacedo"
+approvers:
+  - "@fabianvf"
+  - "@jlanford"
+  - "@dmesser"
+creation-date: 2020-03-10
+last-updated: 2020-03-11
+status: implementable
+---
+
+
+# Ansible/Helm add api command  Proposal for Operator SDK
+
+
+## Release Signoff Checklist
+
+- \[x\] Enhancement is `implementable`
+- \[x\] Design details are appropriately documented from clear requirements
+- \[ \] Test plan is defined
+- \[ \] Accceptance criteria
+- \[ \] User-facing documentation is created
+
+
+
+## Summary
+
+In current scenario, users are not able to add additional APIs for Ansible/Helm based opertators using SDK CLI. This change will make it easier to add additional apis using CLI for Ansible and Helm based operators.
+
```

Comment:
```suggestion
Currently, users are not able to add additional APIs for Ansible/Helm based-operators via SDK CLI tool. This proposal is over provide this feature via the tool. 

```

Reply:
I think I will stick with the original sentencing. It conveys the same.

Reply:
It is not a change, it is a proposal with a propose of providing a new feature to solve a need.

Reply:
it's a proposal to change the existing behavior (the existing behavior being exiting in failure). Adding features is still making a change, I think this phrasing is fine